README.md

```diff
@@ -79,6 +79,45 @@ sudo -u mapped-user aws s3 ls s3://my-custom-mapper/install
 2020-09-11 19:07:41        254 user-role-mapper.properties
 ```
 
+### Integration tests
+- Set the AWS credentials to be available to the environment before running the tests.
+- Like here is one way to do it
+
+```
+export AWS_ACCESS_KEY_ID="ASIASSFAZTPATNBHFC56"
+export AWS_SECRET_ACCESS_KEY="XXXXXXXXX"
+export AWS_SESSION_TOKEN="YYYYYYYYY"
+```
+- To just run unit tests run `mvn test`
+
+### Managed policy union mapper
+- The default mapper allows the mapping to specify different AWS Role, policies etc.
```

Comment:
AWS Role -> IAM Role? same to other places and policies.

Reply:
Done